### ISSUE TITLE
[Bug] [Windows] Use inbox driver when OS install on PVSCSI disk

### ISSUE THREAD

**Issue by Tomorrow9** [vmware/ansible-vsphere-gos-validation#120]
##### SUMMARY

For guest OS which contains the inbox drivers, use inboxed PVSCSI driver instead of downloaded drivers from VMware tools.

##### ISSUE TYPE
- Bug Report

##### ANSIBLE VERSION

```paste below
"ansible [core 2.11.1] ",
        "  config file = /root/ansible-vsphere-gos-validation/ansible.cfg",
        "  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']",
        "  ansible python module location = /usr/lib/python3.9/site-packages/ansible",
        "  ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections",
        "  executable location = /usr/bin/ansible",
        "  python version = 3.9.1 (default, May 19 2021, 10:27:03) [GCC 10.2.0]",
        "  jinja version = 3.0.1",
        "  libyaml = True"
```

##### CONFIGURATION

```paste below
        "CALLBACKS_ENABLED(/root/ansible-vsphere-gos-validation/ansible.cfg) = ['timer']",
        "DEFAULT_CALLBACK_PLUGIN_PATH(/root/ansible-vsphere-gos-validation/ansible.cfg) = ['/root/ansible-vsphere-gos-validation/plugin']",
        "DISPLAY_SKIPPED_HOSTS(/root/ansible-vsphere-gos-validation/ansible.cfg) = False",
        "RETRY_FILES_ENABLED(/root/ansible-vsphere-gos-validation/ansible.cfg) = False"
```

##### ENVIRONMENT / OS

Windows

**Comment by Tomorrow9** [comment]
For Windows Server 2022 and later versions, will not need to download VMware tools iso file to get PVSCSI driver for OS install due to there are inbox drivers.
Also from VMware tools next release, there are some changes on the pvscsi driver folder, which also need to be considered.

**Comment by Tomorrow9** [comment]
Fixed in #217
